scripts/resource.py

```diff
@@ -108,6 +108,9 @@ def cleaner_saas(f):
 def cleaner_elastic(f):
     return f.lower()
 
+def cleaner_outscale(f):
```

Comment:
Please keep 2 lines of space between functions.

Reply:
That should be done,

Thanks for the review